Frontend/areas/administrator/index.html

```diff
@@ -18,130 +18,145 @@
-                        </td>
-                    </tr>
-
-                </tbody>
-            </table>
-
-
-            <form method="post" id="change_user_form"></form>
-
-
-            <table class="table table-hover">
-                <thead class="thead-dark">
-                    <tr>
-                        <th scope="col">Id</th>
-                        <th scope="col">First Name</th>
-                        <th scope="col">Last Name</th>
-                        <th scope="col">Email</th>
-                        <th scope="col">Password</th>
-                        <th scope="col">Role</th>
-                        <th scope="col">Manager</th>
-                        <th scope="col">Update</th>
-                        <th scope="col">Delete</th>
-                    </tr>
-                </thead>
-                <tbody>
-
-                    <tr>
-                        <td>
-                            <input value="1" class="form-control" type="number" name="Id" form="change_user_form" readonly required>
-                        </td>
-                        <td>
-                            <input value="Nazariy" class="form-control" type="text" name="FirstName" form="change_user_form" required />
-                        </td>
-                        <td>
-                            <input value="Tymtsiv" class="form-control" type="text" name="LastName" form="change_user_form" required />
-                        </td>
-                        <td>
-                            <input value="nazar@gmail.com" class="form-control" type="email" name="Email" form="change_user_form" required />
-                        </td>
-                        <td>
-                            <input value="1111" class="form-control" type="password" name="Password" form="change_user_form" required />
-                        </td>
-                        <td>
-                            <select class="form-control" name="Role" form="change_user_form" required />
-                            <option selected>Resident</option>
-                            <option>Manager</option>
-                        </td>
-                        <td>
-                            <select class="form-control" name="Manager" form="change_user_form" />
-                            <option>Nazar</option>
-                            <option>Taras</option>
-                            <option selected>Ivan</option>
-                        </td>
-                        <td>
-                            <input value="Submit" class="form-control" type="submit" name="Update" form="change_user_form" />
-                        </td>
-                        <td>
-                            <input value="Delete" class="form-control" type="submit" name="Delete" form="change_user_form" />
-                        </td>
-                    </tr>
-
-
-
-                </tbody>
-            </table>
+            <div>
```

Comment:
wait.... you changed it only in **frontend** project? it takes me a 15 minutes to figure it out.

Reply:
yes, but I have added it to view just now.